### PR TITLE
added a test function

### LINKED ISSUE #9: Add test for model fitting

### PR DESCRIPTION
**Pull request by rossellabelleli**

### CODE REVIEW

File: tests/testthat/test-fit_models.R [hunk 1]
```diff
@@ -0,0 +1,4 @@
+test_that("fit model works as expected", {
+  result <- fit_models()
```
Comment (MikeJSeo) [inline tests/testthat/test-fit_models.R:2]: add more code

Comment (MikeJSeo) [inline tests/testthat/test-fit_models.R:2]:
```suggestion
  result <- fit_models()
  print("hi")
```

**Review by MikeJSeo — COMMENTED** [review]
great work thanks rossella. few comments

File: tests/testthat/test-fit_models.R (hunk 1 above)
Comment (rossellabelleli) [reply tests/testthat/test-fit_models.R:2]: I will

**Pull request closed without merging**